recipes/example/meta.yaml

```diff
@@ -1,83 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Can you restore the example, the CI will ignore any recipes which aren't part of your PR.

Reply:
Done.